src/test/java/com/mentoring/framework/utils/InitData.java

```diff
@@ -0,0 +1,44 @@
+package com.mentoring.framework.utils;
+
+import com.mentoring.framework.Config;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.http.client.ClientProtocolException;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.HttpClientBuilder;
+
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+
+@Slf4j
+public final class InitData {
+
+    private InitData() {
+    }
+
+    public static void registerUser(String username, String password, String email) {
+        try {
+            HttpClient client = HttpClientBuilder.create().build();
+
+            HttpPost postRequest = new HttpPost(Config.getApplicationUrl() + "/user");
+
+            StringEntity input = new StringEntity(
```

Comment:
This is where RestAssured would look much more readable and clean. With RestAssured you can also declare the expected response code, so your tests can fail-fast if the rest call fails.